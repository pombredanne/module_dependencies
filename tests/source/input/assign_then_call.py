import nltk

sentence_tokenizer = 12


class MyObject:
    pass


obj = MyObject()

if sentence_tokenizer:
    sentence_tokenizer = sentence_tokenizer
else:
    obj.sentence_tokenizer = nltk.tokenize.sent_tokenize

obj.sentence_tokenizer()
