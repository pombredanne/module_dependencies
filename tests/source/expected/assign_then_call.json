{
    "dependencies": [
        "nltk.tokenize.sent_tokenize"
    ],
    "imports": [
        "nltk"
    ]
}
